Update contributors guide (#3332)
Chatterino/chatterino2: resources/contributors.txt

# Adding yourself? Copy and paste this template at the bottom of this file and fill in the fields in a PR!
# Name | Link | Avatar (Loaded as a resource) | Title (description of work done).

> # Avatar should be located in avatars/ directory. Its size should be 128x128 (get it from https://avatars.githubusercontent.com/username?size=128).

Reviewer: Why is this changed? I don't see a point in doing so, while old method still works and will continue to do so with a fancy redirect.

Reviewer 2: ```suggestion
# Avatar should be located in avatars/ directory. Its size should be 128x128 (get it from https://github.com/username.png?size=128).
```

> # Avatar should be located in avatars/ directory. It's size should be 128x128 (get it from https://avatars.githubusercontent.com/username?size=128).

Reviewer 3: ```suggestion
# Avatar should be located in avatars/ directory. Its size should be 128x128 (get it from https://avatars.githubusercontent.com/username?size=128).
```

"Its" is the possessive form of "it" - whereas "It's" is a contraction of "it is" :)

> # Make sure to reduce avatar's size as much as possible with tool like pngcrush or optipng (if the file is in png format) and run `./generate_resources.py`.

Reviewer 2: ```suggestion
# Make sure to reduce avatar's size as much as possible with tool like pngcrush or optipng (if the file is in png format) and run ./generate_resources.py
```

Removed the backticks as this is not a markdown file, and removed the dot to make it easier to copy the script name
